tgui/packages/tgui/interfaces/CTFPanel.js

```diff
@@ -0,0 +1,46 @@
+import { useBackend } from '../backend';
+import { Button, Section, Stack } from '../components';
+import { Window } from '../layouts';
+
+export const CTFPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const teams = data.teams || [];
+  const enabled = data.enabled || [];
+  return (
+    <Window
+      title="CTF Panel"
+      width={700}
+      height={600}>
+      <Window.Content scrollable>
+        <Section title={enabled} textAlign="center">
+          {teams.map(team => (
+            <Section
+              key={team.name}
+              title={team.color + ' Team' + ' (Score:' + team.score + ') (Members:' + team.team_size + ')'}>
```

Comment:
```suggestion
              title={`${team.color} Team (${team.score} points) (${team.team_size} members)'}>
```

Though, the best UI here would be showing a little dude instead of the word members, think of how the orbit menu does it. You don't have too, but it would just look the best.